resources/content_list.ts

```diff
@@ -393,6 +393,9 @@ const contentList: (ZoneIdType)[] = [
 
   // Masked Carnivale
   ZoneId.AllsWellThatStartsWell,
+
+  // Treasure Map Dungeons
+  ZoneId.TheExcitatron6000,
```

Comment:
Also, I think these things shouldn't be added to the content_list at this point.

I think really there needs to be some additional feature to the coverage where miscellaneous things (hunts, pvp, maps, carnivale) can be added in a separate list.  But for now, I think it's more confusing than helpful, if that makes sense.

Reply:
sounds good, reverted the changes to content_list